src/main/java/org/opentripplanner/routing/edgetype/rentedgetype/DropBikeEdge.java

```diff
@@ -0,0 +1,19 @@
+package org.opentripplanner.routing.edgetype.rentedgetype;
+
+import org.opentripplanner.routing.bike_rental.BikeRentalStation;
+import org.opentripplanner.routing.core.vehicle_sharing.VehicleDescription;
+import org.opentripplanner.routing.graph.Vertex;
+
+public class DropBikeEdge extends DropoffVehicleEdge {
```

Comment:
może komentarz, że to jest tylko do oddawania rowerów i jest tylko w wierzchołkach przy stacjach rowerowych